kibble/render/render.go

```diff
@@ -125,39 +123,13 @@ func Render(sourcePath string, buildPath string, cfg *models.Config) int {
 		// set the template view
 		renderer.view = models.CreateTemplateView(routeRegistry, T, &ctx, sourcePath)
 
-		// render static files
```

Comment:
I have a feeling this removal is going to break every single kibble site. As none of the static files in the root will be rendered (signup.html.jet, signin.html.jet, etc).

Reply:
Yes had the same fear.

Reply:
This adds in an implicit route to ensure we get the same behaviour
https://github.com/shift72/kibble/pull/72/files#diff-f4691f91e3d982c814c71a8bac619215dc631ee5f3d482e9661ce33bbd5696a4R184-R190